.env

```diff
@@ -0,0 +1,35 @@
+OPEN_RUNTIMES_EXECUTOR_ENV=development
+OPEN_RUNTIMES_EXECUTOR_WORKER_PER_CORE=6
+OPEN_RUNTIMES_EXECUTOR_STORAGE_DEVICE=Local
+OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_ACCESS_KEY=
+OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_SECRET=
+OPEN_RUNTIMES_EXECUTOR_STORAGE_S3_REGION=us-east-1
```

Comment:
Add a TODO:
Move to use DSN instead of all these storage env vars. A single var could replace all  `OPEN_RUNTIMES_EXECUTOR_STORAGE_*`

Reply:
https://github.com/open-runtimes/executor/pull/2

Created a PR for it here

Reply:
Resolving this discussion as it should be discussed in new PR.